doc/man3/OSSL_STORE_expect.pod

```diff
@@ -0,0 +1,77 @@
+=pod
+
+=head1 NAME
+
+OSSL_STORE_expect,
+OSSL_STORE_supports_search,
+OSSL_STORE_find
+- Specify what object type is expected
+
+=head1 SYNOPSIS
+
+ #include <openssl/store.h>
+
+ int OSSL_STORE_expect(OSSL_STORE_CTX *ctx, int expected_type);
+
+ int OSSL_STORE_supports_search(OSSL_STORE_CTX *ctx, int criterion_type);
+ 
+ int OSSL_STORE_find(OSSL_STORE_CTX *ctx, OSSL_STORE_SEARCH *search);
+
+=head1 DESCRIPTION
+
+OSSL_STORE_expect() helps applications filter what OSSL_STORE_load() returns
+by specifying a B<OSSL_STORE_INFO> type.
+For example, if C<file:/foo/bar/store.pem> contains several different objects
+and only the certificates are interesting, the application can simply say
+that it expects the type B<OSSL_STORE_INFO_CERT>.
```

Comment:
Are all the types listed somewhere in the docs? If so a reference to that page would be good. If not - perhaps they should be?

Reply:
Yes.  [`doc/man3/OSSL_STORE_INFO.pod\SUPPORTED OBJECTS`](https://github.com/openssl/openssl/blob/master/doc/man3/OSSL_STORE_INFO.pod#supported-objects)

I'll a reference here.

Reply:
Fixed, with some explaining text